### ISSUE TITLE
When I Add in Drapes to ZoneShades, Make sure to Think of a Workflow for Electrochromic Glazing

### ISSUE THREAD

**Issue by chriswmackey** [mostaphaRoudsari/Honeybee#300]
As per this discussion:
http://www.grasshopper3d.com/group/ladybug/forum/topics/error-message-when-i-use-the-indoor-view-factor-calculator?commentId=2985220%3AComment%3A1240400&xg_source=msg_com_gr_forum

**Comment by chriswmackey** [comment]
I can finally close out this issue:
https://github.com/mostaphaRoudsari/Honeybee/commit/180bac5d289e46c148ecbe61809e0c26324c301f
Phew!

**Comment by chriswmackey** [comment]
Also, @ayezioro ,
Let me know if you agree with the implementation,  I realized that all of the properties of a shade material coincide nicely with a window construction in an "on" electrochromic state.  If you feel there are certain key features that are missing with this workflow, let me know and we can strategize a better way to implement it.
-Chris

**Comment by ayezioro** [comment]
Oh, i will :-)
Just get the email with these news and i'm eager to test this ASAP. Coincidentialy i had a class today where we deal with  blinds. Almost perfect timing.
Thanks!!!
-A.

**Comment by ayezioro** [comment]
Hi Chris,
Started to check, but in the process i get an IDF than doesn't contain any of the definitions: No windows (at all), no shades, no blinds, no materials etc, related to the updated compoent.
I just run the hydra example.
Can you confirm there is an issue there?
Thanks,
-A.

**Comment by chriswmackey** [comment]
@ayezioro ,
Thanks for checking this so quickly and very sorry for the late reply.  I had messed things up so badly that I did not even notice that I had messed up.  I guess the simulation was running so well because none of my objects were actually being written.

Needless to say, everything should be fixed after this commit:
https://github.com/mostaphaRoudsari/Honeybee/commit/ed6cf17f9bb503daed381231c4332d7375026f6a

Also, I updated the example file, which is now writing the objects as intended:
http://hydrashare.github.io/hydra/viewer?owner=chriswmackey&fork=hydra_2&id=EnergyPlus_Window_Shade_Generator

Thanks again, Abraham. Your diligence with checking such things is honorable!
-Chris
